README.md

```diff
@@ -1,5 +1,9 @@
 # AOZ Voluntar Platform
 
+master: [![Build Status](https://travis-ci.org/panter/aoz-003.svg?branch=master)](https://travis-ci.org/panter/aoz-003)
+develop: [![Build Status](https://travis-ci.org/panter/aoz-003.svg?branch=develop)](https://travis-ci.org/panter/aoz-003)
```

Comment:
We only need the build status for development, because, the only branch that gets merged into master is development